userguide/payment/includes/payment-flows.adoc

```diff
@@ -32,9 +32,9 @@ The result of the API call will depend on the type of HPP integration:
 
 Note that after calling *buildFormDescriptor*, the payment may or may not exist in Kill Bill: this will depend on the plugin. If it does however, it will be most likely in a **PENDING** state.
 
-Here are a few common scenarii:
```

Comment:
`scenarii` is incorrect? Or maybe just too pedantic?

Reply:
It is indeed pedantic; not commonly used anymore.